tests/functional/modules/test_zos_archive.py

```diff
@@ -0,0 +1,925 @@
+                                        ),
+                                        mode=dest_mode)
+        print(archive_result.contacted.values())
+        stat_dest_res = hosts.all.stat(path=dest)
+        for result in archive_result.contacted.values():
+            print(result)
+            assert result.get("failed", False) is False
+            assert result.get("changed") is True
+            assert result.get("dest_state") == expected_state
+            for stat_result in stat_dest_res.contacted.values():
+                print(stat_result)
+                assert stat_result.get("stat").get("exists") is True
+                assert stat_result.get("stat").get("mode") == dest_mode
+                
+    finally:
+        hosts.all.file(path=f"{USS_TEMP_DIR}", state="absent")
+
+
+@pytest.mark.uss
+@pytest.mark.parametrize("format", USS_FORMATS)
+def test_uss_single_archive_with_force_option(ansible_zos_module, format):
+    try:
+        hosts = ansible_zos_module
+        hosts.all.file(path=f"{USS_TEMP_DIR}", state="absent")
+        hosts.all.file(path=USS_TEMP_DIR, state="directory")
+        set_uss_test_env(hosts, USS_TEST_FILES)
+        dest = f"{USS_TEMP_DIR}/archive.{format}"
+        expected_state = STATE_ARCHIVED
+        archive_result = hosts.all.zos_archive( path=list(USS_TEST_FILES.keys()),
+                                        dest=dest,
+                                        format=dict(
+                                            name=format
+                                        ))
+        print(archive_result.contacted.values())
+
+        for result in archive_result.contacted.values():
+            print(result)
+            assert result.get("failed", False) is False
+            assert result.get("changed") is True
+
+        archive_result = hosts.all.zos_archive( path=list(USS_TEST_FILES.keys()),
+                                        dest=dest,
+                                        format=dict(
+                                            name=format
+                                        ))
+        
+        for result in archive_result.contacted.values():
+            print(result)
+            assert result.get("failed", False) is True
+            assert result.get("changed") is False
+        
+        set_uss_test_env(hosts, USS_TEST_FILES)
+        archive_result = hosts.all.zos_archive( path=list(USS_TEST_FILES.keys()),
+                                        dest=dest,
+                                        format=dict(
+                                            name=format
+                                        ),
+                                        force=True,)
+
+        for result in archive_result.contacted.values():
+            print(result)
+            assert result.get("failed", False) is False
+            assert result.get("changed") is True
+            assert result.get("dest_state") == expected_state
+
```

Comment:
Did ansible not complain about this double space?  I thought single-space was mandatory, and double-spaces were between functions.  If Ansible didn't complain, I won't.

Reply:
I ran ansible lint and sanity , ansible had no issues on the code i reviewed, closing